store/sqlstore/user_store.go

```diff
@@ -367,61 +386,42 @@ func (s SqlUserStore) GetEtagForAllProfiles() store.StoreChannel {
 func (us SqlUserStore) GetAllProfiles(options *model.UserGetOptions) store.StoreChannel {
 	isPostgreSQL := us.DriverName() == model.DATABASE_DRIVER_POSTGRES
 	return store.Do(func(result *store.StoreResult) {
-		var users []*model.User
-		offset := options.Page * options.PerPage
-		limit := options.PerPage
+		query := usersQuery.
+			OrderBy("u.Username ASC").
+			Offset(options.Page * options.PerPage).Limit(options.PerPage)
 
-		searchQuery := `
-			SELECT * FROM Users
-			WHERE_CONDITION
-			ORDER BY Username ASC LIMIT :Limit OFFSET :Offset
-		`
+		query = applyRoleFilter(query, options.Role, isPostgreSQL)
 
-		parameters := map[string]interface{}{"Offset": offset, "Limit": limit}
-		searchQuery = substituteWhereClause(searchQuery, options, parameters, isPostgreSQL)
+		if options.Inactive {
+			query = query.Where("u.DeleteAt != 0")
+		}
 
-		if _, err := us.GetReplica().Select(&users, searchQuery, parameters); err != nil {
+		var users []*model.User
+		if _, err := us.GetReplica().Select(&users, query.String(), query.Args()); err != nil {
 			result.Err = model.NewAppError("SqlUserStore.GetAllProfiles", "store.sql_user.get_profiles.app_error", nil, err.Error(), http.StatusInternalServerError)
-		} else {
-
-			for _, u := range users {
-				u.Sanitize(map[string]bool{})
-			}
+			return
+		}
 
-			result.Data = users
+		for _, u := range users {
+			u.Sanitize(map[string]bool{})
 		}
+
+		result.Data = users
 	})
 }
 
-func substituteWhereClause(searchQuery string, options *model.UserGetOptions, parameters map[string]interface{}, isPostgreSQL bool) string {
-	whereClause := ""
-	whereClauses := []string{}
-	if options.Role != "" {
-		whereClauses = append(whereClauses, getRoleFilter(isPostgreSQL))
-		parameters["Role"] = fmt.Sprintf("%%%s%%", options.Role)
-	}
-	if options.Inactive {
-		whereClauses = append(whereClauses, " Users.DeleteAt != 0 ")
+func applyRoleFilter(query *querybuilder.Query, role string, isPostgreSQL bool) *querybuilder.Query {
+	if role == "" {
+		return query
 	}
 
-	if len(whereClauses) > 0 {
```

Comment:
All of this magic is now unnecessary since we have no need to remove string constants from the queries that are unused.